RebarDistribution.py

```diff
@@ -72,9 +72,22 @@ def accept(self):
             self.ExpandingLength,
         )
 
-    def setupUi(self):
+    def reject(self):
+        # self.CustomSpacing = ""
+        pass
+
+    def setupUi(self, CustomSpacing):
         # Connect Signals and Slots
         self.form.buttonBox.accepted.connect(self.accept)
+        self.form.buttonBox.rejected.connect(self.reject)
+        if CustomSpacing:
+            spacingList = getupleOfCustomSpacing(CustomSpacing)
+            self.form.amount1.setValue(spacingList[0][0])
+            self.form.spacing1.setText(str(spacingList[0][1]))
+            self.form.amount2.setValue(spacingList[1][0])
+            self.form.spacing2.setText(str(spacingList[1][1]))
+            self.form.amount3.setValue(spacingList[2][0])
+            self.form.spacing3.setText(str(spacingList[2][1]))
```

Comment:
Before accessing values, check length of spacinglist.
Because, in case, Reinforcement is created through scripting and `rebar.CustomSpacing = "2@50.0+1@100.0"`
Then, it will give `IndexError`

Reply:
So if the value of a segment is not given, should I leave it as default of make it zero?
For example, if `rebar.CustomSpacing = "2@50.0+1@100.0"` then should I make `amount3` and `spacing3` equal to 0?

Reply:
Yes, make that 0